### DISCUSSION TITLE
Interacting with code-server or VS Code Server

### DISCUSSION THREAD

**Discussion by zeidel** [blinksh/blink#1656] [Q&A]
Is it possible to connect Blink Code to the existing remote code-server (https://github.com/coder/code-server) instance or VS Code Server (https://code.visualstudio.com/docs/remote/vscode-server)? 

Right now, instead of Blink Code I'm using SSH tunnel, which connects to the code-server and after that, in any browser, I can use it. But maybe it's possible to get rid of it (additional ssh tunnel+browser) and use Blink Code for that?

The use case is not to simply open folders with Code, but also vscode workspaces and access all configured extensions and configurations for existing workspaces

Tnx

**Comment by TobiaszCudnik** [comment]
Yes.

- bottom left corner
- Connect to Tunnel
- auth github
- on the remote machine `code tunnel` (download CLI)
- auth github
- Connect to Tunnel

More info:
https://code.visualstudio.com/docs/remote/tunnels#_using-the-code-cli
